addons/website/static/src/snippets/s_image_gallery/002.scss

```
@@ -0,0 +1,330 @@
+
+        &.o_spc-medium div.o_masonry_col {
+            padding: 0 $spacer;
+
+            > img, > a > img, > .media_iframe_video {
+                margin-bottom: $spacer * 2 !important;
+            }
+        }
+
+        &.o_spc-big div.o_masonry_col {
+            padding: 0 ($spacer * 1.5);
+
+            > img, > a > img, > .media_iframe_video {
+                margin-bottom: $spacer * 3 !important;
+            }
+        }
+    }
+
+    &.o_nomode {
+        &.o_spc-none .row > div {
+            padding-top: 0;
+            padding-bottom: 0;
+        }
+
+        &.o_spc-small .row > div {
+            padding-top: $spacer * .5;
+            padding-bottom: $spacer * .5;
+        }
+
+        &.o_spc-medium .row > div {
+            padding-top: $spacer;
+            padding-bottom: $spacer;
+        }
+
+        &.o_spc-big .row > div {
+            padding-top: $spacer * 1.5;
+            padding-bottom: $spacer * 1.5;
+        }
+    }
+
+    &:not(.o_slideshow) {
+        img {
+            cursor: pointer;
+        }
+    }
+
+    &.o_slideshow {
+        --o-carousel-indicators-dots-size: .75rem;
+        --o-carousel-controllers-size: #{map-get($spacers, 5)};
+
+        .carousel {
+            &, .carousel-inner {
+                height: var(--snippet-preview-height, 100%);
+            }
+
+            .carousel-item.active, .carousel-item-next, .carousel-item-prev {
+                display: flex;
+                align-items: center;
+                height: var(--snippet-preview-height, 100%);
+            }
+
+            .media_iframe_video {
+                // 70% because the carousel controls are each 15% and the user
+                // must be able to click on the video buttons.
+                width: 70%;
+                height: 100%;
+            }
+        }
+
+        > .container, > .container-fluid, > .o_container_small {
+            height: var(--snippet-preview-height, 100%);
+        }
+
+        > .container-fluid {
+            padding: 0;
+        }
+
+        .o_carousel_controllers {
+            display: contents;
+        }
+
+        .carousel-indicators {
+            align-items: center;
+        }
+
+        .carousel-control-prev, .carousel-control-next {
+            border-radius: $border-radius;
+        }
+
+        .carousel-control-prev-icon, .carousel-control-next-icon {
+            width: var(--o-carousel-controllers-size);
+            height: var(--o-carousel-controllers-size);
+            background-size: 50%;
+        }
+
+        // Arrows - Default
+        &.s_image_gallery_arrows_default {
+            .carousel-control-prev, .carousel-control-next {
+                &:hover {
+                    background: rgba($carousel-dark-indicator-active-bg, .2);
+                }
+            }
+            .carousel-dark {
+                .carousel-control-prev, .carousel-control-next {
+                    &:hover {
+                        background: rgba($white, .2);
```

Shouldn't this be `$carousel-indicator-active-bg` 🤔 (it's white by default)
```suggestion
                        background: rgba($carousel-indicator-active-bg, .2);
```